client/local_client.go

```diff
@@ -57,7 +58,12 @@ func (c *client) ResyncConfig(items ...proto.Message) error {
 		txn.Put(key, item)
 	}
 
-	return txn.Commit(context.Background())
+	ctx := context.Background()
+	_, withDataSrc := orch.DataSrcFromContext(ctx)
```

Comment:
Will this even return `withDataSrc=true` if we use `context.Background()` here?

Reply:
if condition added by mistake - removed